nntrainer/layers/common_properties.h

```diff
@@ -137,65 +137,75 @@ class Standardization : public nntrainer::Property<bool> {
 };
 
 /**
- * @brief RAII class to define the connection spec
+ * @brief RAII class to define the connection
  *
  */
-class ConnectionSpec {
+class Connection {
 public:
-  static std::string NoneType;
-
   /**
-   * @brief Construct a new Connection Spec object
+   * @brief Construct a new Connection object
    *
-   * @param layer_ids_ layer ids that will be an operand
-   * @param op_type_ operator type
+   * @param layer_name layer identifier
    */
-  ConnectionSpec(const std::vector<Name> &layer_ids_,
-                 const std::string &op_type_ = ConnectionSpec::NoneType);
+  Connection(const std::string &layer_name, unsigned int idx);
```

Comment:
if the idx is for tensor, then could you leave it in a comment?

Reply:
As connection is being extracted out, I'll make a change around https://github.com/nnstreamer/nntrainer/pull/1738/commits/889551a34b1d2b58dbbd83d41fcdff7af27f4552 :)